main/src/modules/Jvm.scala

```diff
@@ -376,7 +376,7 @@ object Jvm {
                               shebang: Boolean = false) = {
     universalScript(
       shellCommands =
-        s"""exec java ${jvmArgs.mkString(" ")} $$JAVA_OPTS -cp "${shellClassPath.mkString(":")}" $mainClass "$$@"""",
+        s"""exec java ${jvmArgs.mkString(" ")} $$JAVA_OPTS -cp "${shellClassPath.mkString(":")}" '$mainClass' "$$@"""",
```

Comment:
Yeah, makes sense. But we the different quoting sign?

Reply:
Not sure I follow. It has to be single quotes here, as double quotes would still expand any dollar sign.

Reply:
By bad. You're completely right.